sycl/include/sycl/detail/os_util.hpp

```diff
@@ -12,7 +12,10 @@
 
 #include <sycl/detail/export.hpp> // for __SYCL_EXPORT
 
-#include <cstdlib>    // for size_t
+#include <cstdlib> // for size_t
+#ifdef _WIN32
+#include <filesystem>
+#endif
```

Comment:
This header is public interface and included to user's application. I don't believe that DPC++ should include `filesystem` header to user's code. This is required only for the runtime library, so it must be an internal detail of the runtime library. Do you agree?

Reply:
I agree, fixed to not include that in user headers.